apollo-router/src/protocols/websocket.rs

```diff
@@ -267,15 +267,9 @@ where
         let first_non_ping_payload = async {
             loop {
                 match stream.next().await {
-                    Some(Ok(ServerMessage::Ping { payload })) => {
-                        // we don't mind an error here
-                        // because it will fall through the error below
-                        // if we haven't been able to properly get a ConnectionAck within the `CONNECTION_ACK_TIMEOUT`
-                        let _ = stream
-                            .send(ClientMessage::Pong {
-                                payload: payload.map(|p| p.into()),
-                            })
-                            .await;
```

Comment:
Tested this manually and now I don't think its safe to remove - will revisit the test to understand why it was receiving 2 pongs and restore this